Gemfile.base

```diff
@@ -206,7 +206,6 @@ group :test do
   gem 'webmock', '~> 2.3.2'
   gem 'launchy'
   gem 'mechanize'
-  gem 'parallel_tests', '~> 2.3'
```

Comment:
Can this affect https://github.com/3scale/porta/tree/master/lib/tasks/integration.rake?

Reply:
I don't know how that works but that is for CircleCI, right? and CircleCI is passing all tests in this PR because it does not use parallel_tests. In fact we are not using it anymore for anything.

Reply:
We can remove it it was for jenkins
And migrating to Rails 6 we can have it back natively

Reply:
@hallelujah What do u mean? part of the changes u requested before were in that file for CircleCI, so I cannot remove the whole file, can I? :confused: